tests/test_sample_rayjob_yamls.py

```diff
@@ -47,7 +47,7 @@
         test_cases.addTest(GeneralTestCase('runtest', addEvent))
 
     # Execute all testsCRs
-    runner = unittest.TextTestRunner()
+    runner = unittest.TextTestRunner(failfast=True)
```

Comment:
Could we also apply the same logic to `tests/test_sample_rayservice_yamls.py`? The file `tests/test_sample_rayservice_yamls.py` uses pytest, and I found `--exitfirst`, which is similar to `failfast` in unittest.

Reply:
Thanks for pointing that out! This is a good catch, and I would follow up to complete.